### PR TITLE
set UNLEASH_URL and manage openshift route

### PR DESCRIPTION
**Pull request by chassing**
see APPSRE-6023

### CODE REVIEW

File: openshift/unleash.yaml [hunk 1]
```diff
@@ -148,6 +150,29 @@ objects:
         name: ${identifier}
     selector:
       app: ${identifier}
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    annotations:
+      cert-manager.io/issuer-kind: ClusterIssuer
+      cert-manager.io/issuer-name: letsencrypt-prod-http
```
Comment (rporres) [inline openshift/unleash.yaml:158]: why adding this annotations now? I'm just worried that in a moment we have both cert-manager in the cluster AND openshift-acme in the namespace...

Comment (chassing) [reply openshift/unleash.yaml:158]: I took it form the current a-i route.

Comment (chassing) [reply openshift/unleash.yaml:158]: cert-manager is the new kid in town (https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/43592/diffs)

Comment (chassing) [reply openshift/unleash.yaml:158]: I've removed the `openshift-acme` annotation.

**Review by rporres — APPROVED** [review]

**Pull request merged**